pkg/sql/schemachanger/scplan/internal/rules/current/dep_alter_column_type.go

```diff
@@ -52,4 +52,21 @@ func init() {
 			}
 		},
 	)
+
```

Comment:
ah i guess i had the same question about deprules as you did now :-); do we want to preserve 24.3's rules and then have this be in "current"?